### PR TITLE
StringExtensions Unit Tests

### PR DESCRIPTION
**Pull request by agriffard**
Testing GitHub Copilot to generate Unit Tests.

### CODE REVIEW

File: test/OrchardCore.Abstractions.Tests/Modules/StringExtensionsTests.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
+using System;
+using Xunit;
+
+namespace OrchardCore.Modules
+{
+    public class StringExtensionsTests
+    {
+        [Fact]
```
Comment (hishamco) [inline test/OrchardCore.Abstractions.Tests/Modules/StringExtensionsTests.cs:8]: We could use `Theory` for both test case 1 & 3

File: test/OrchardCore.Abstractions.Tests/Modules/StringExtensionsTests.cs [hunk 2]
```diff
@@ -0,0 +1,79 @@
+using System;
+using Xunit;
+
+namespace OrchardCore.Modules
+{
+    public class StringExtensionsTests
+    {
+        [Theory]
+        [InlineData(new byte[]{ 10, 20, 30 }, "0A141E")]
+        public void ToHexString_ReturnsCorrectHexString(byte[] bytes, string expected)
```
Comment (hishamco) [inline test/OrchardCore.Abstractions.Tests/Modules/StringExtensionsTests.cs:10]:
```suggestion
        [InlineData(new byte[0], "")]
        public void ToHexString_ReturnsCorrectHexString(byte[] bytes, string expected)
```

File: test/OrchardCore.Abstractions.Tests/Modules/StringExtensionsTests.cs [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        {
+            // Arrange
+            byte[] bytes = null;
+
+            // Act & Assert
+            Assert.Throws<ArgumentNullException>(() => bytes.ToHexString());
+        }
+
+        [Fact]
+        public void ToHexString_EmptyInput_ReturnsEmptyString()
```
Comment (hishamco) [inline test/OrchardCore.Abstractions.Tests/Modules/StringExtensionsTests.cs:32]: Now you can remove this test after the above change

**Review by hishamco — CHANGES_REQUESTED** [review]

File: test/OrchardCore.Abstractions.Tests/Modules/StringExtensionsTests.cs (hunk 1 above)
Comment (agriffard) [reply test/OrchardCore.Abstractions.Tests/Modules/StringExtensionsTests.cs]: Ok, using `Theory` in test case 1 & 3.

**Review by hishamco — APPROVED** [review]

**Pull request merged**